package.json

```diff
@@ -9,16 +9,16 @@
     "test": "./node_modules/.bin/mocha"
   },
   "dependencies": {
-    "ip": "~0.3.1",
-    "moment": "~2.8.1"
+    "ip": "~1.1.2",
+    "moment": "~2.12.0"
   },
   "devDependencies": {
-    "async": "~0.9.0",
-    "express": "~4.9.5",
-    "lodash": "~2.4.1",
-    "mocha": "~1.21.4",
-    "redis": "~0.12.1",
-    "should": "~4.0.4",
-    "supertest": "~0.13.0"
+    "async": "~2.0.0-rc.2",
```

Comment:
`~2.0.0` actually doesn't pick `~2.0.0-rc.3` because it's a pre-release version (anything after the `-` is a pre-release).
You can have a play at http://semver.npmjs.com/

But very good point that maybe we shouldn't be using an `rc` version! Updated to use the latest stable `1.5.2`.